virus/Anas_Amicus.md

```diff
@@ -0,0 +1,203 @@
+# Anas Amicus: Ducking into the "Exemplar" Obelisk
+written by: [Izumi Ando](https://github.com/yourname *optional link*)
```

Comment:
If you're not linking to your github account, delete the link, else add the link.